.docker/Dockerfile

```diff
@@ -91,11 +113,6 @@ RUN docker-php-ext-install -j$(nproc) zip
 FROM ${BASE}-php-ext-base AS php-ext-xsl
 RUN docker-php-ext-install -j$(nproc) xsl
 
-# php extension redis
```

Comment:
Just a double-check question: Does Kimai support Redis in general?
If yes, why not keep Redis?
If not, why has this been added in the first place? (Any idea?)

Reply:
I actually don't know. I played around with it 2-3 years ago, but ditched it, because it needs custom configurations and for > 90% of all installations there is no benefit. 
If someone wants to use it, that person should build its own image.